docs/guide-using-an-ide-for-backend.md

```diff
@@ -161,9 +160,25 @@ Sadly, it's not possible to generate multiple files from one template, so the Me
 * Run, Edit configurations..., `+`, Tomcat, Local.
 * Call it `molgenis-app [exploded]`
 * (Add and) select your Tomcat installation
-* VM options: `-Dmolgenis.home=<path to dir containing molgenis-server.properties> -Xmx4g -Des.discovery.zen.ping.multicast.enabled=false -Des.network.host=localhost`
+* Select the 'Startup/Connection' tab
```

Comment:
If you copy an old run configuration, you should empty the `VM options` field in the `Server` tab. If you don't, there is an unremovable `JAVA_OPTS` variable in the `Startup/Connection` tab.